pkg/bpf/map.go

```diff
@@ -643,7 +643,7 @@ func (m *Map) DeleteWithErrno(key MapKey) (error, syscall.Errno) {
 	_, errno = deleteElement(m.fd, key.GetKeyPtr())
 
 	if errno != 0 {
-		err = fmt.Errorf("Unable to delete element from map: %s", err)
+		err = fmt.Errorf("Unable to delete element from map %s: %s", m.name, errno)
```

Comment:
can you change this to `errno.Error()` - it will give the error string that corresponds to `errno`.

Reply:
Thanks. (updated)